src/pubsub/nodetree_tree.erl

```diff
@@ -49,6 +49,16 @@
          get_subnodes/3, create_node/6,
          delete_node/2]).
 
+-define(MOD_PUBSUB_DB_BACKEND, mod_pubsub_db_backend).
```

Comment:
I think this code would look better and be more 'greppable' if we skip this macro and use the module names directly. However, it's not a blocker, this is temporary anyway.